t/plugins.t

```diff
@@ -16,6 +16,15 @@ for my $file (<./plugins/*.json>) {
         ok !$@, "valid JSON";
         ok $json->{source}, "has source";
         ok $json->{description}, "has description";
+
+        my $valid = like $json->{source}, qr|\A[-_.a-zA-Z0-9]+/[-_.a-zA-Z0-9]+\Z|, 'source is valid';
+        # test only safe sources
+        if ($valid) {
+            # curl -f exists with 0 when request succeeded
```

Comment:
Probably a typo of `exits`.

Reply:
Oh…
https://github.com/mackerelio/plugin-registry/pull/10/commits/740a0c7490b4596ffdb95150167637f4ee74011b